vignettes/netCDF_Functionality.Rmd

```diff
@@ -0,0 +1,400 @@
+---
+title: "Demonstration of netCDF functionality in rSW2analysis"
+author: "Caitlin M. Andrews"
+date: "`r Sys.Date()`"
+output:
+  rmarkdown::html_vignette:
+    toc: true
+    toc_depth: 2
+vignette: >
+  %\VignetteIndexEntry{Demonstration of netCDF functionality in rSW2analysis}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+# Overview
+
+This vignette is an overview of how to use the functions in the rSW2analysis package to
+convert arrays of rSOILWAT2 and SOILWAT2 output data into netCDFs that adhere to
+[CF conventions](http://cfconventions.org/Data/cf-conventions/cf-conventions-1.8/cf-conventions.html#_orthogonal_multidimensional_array_representation).
+
+Additionally, this document describes the Dryland Ecology Lab's standards and
+conventions in regards to naming, metadata, and netCDF structure.
+
+# Input Data Format and Structure
+
+## Variable Data
+
+The netCDF functions in this package expect an array, in which each row represents
+data for a site. Columns represent one of three things:
+
+* multiple different variables for one site
+* one variable with multiple measurements (time series)
+* one variable with multiple depth measurements
+
+```{r, create1d, results = 'hide'}
+   # load rSW2analysis pacckage
+    library(rSW2analysis)
+
+   # dummy data for 10 sites, 10 data points (could be 10 vars, 10 times, or 10 depths)
+   someData <- rnorm(100, 7, 30)
+   data1d <- array(someData, c(10, 10))
```

Comment:
I wasn't interpreting much into this variable name, but then `data3d` suggests that it contains 3-dimensional data. So, why is the 2-dimensional array called `data1d` and not `data2d`?

Reply:
changed